src/products/review/README.md

```diff
@@ -1,4 +1,4 @@
 # Review process
 
-* [How to review strings in Firefox desktop and Firefox for Android](review.md)
-* [How to manage sign-offs for Firefox desktop and Firefox for Android](signoffs.md)
+* [How to review strings in Firefox desktop](review.md)
+* [How to manage sign-offs for Firefox desktop](signoffs.md)
```

Comment:
I agree that we should keep the content of the signoffs page unchanged. For that reason, I would also keep *Firefox for Android* in the link here.